test/a11y.html

```diff
@@ -6,11 +6,12 @@
 
 		<title>d2l-offscreen test</title>
 
-		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-		<script src="../../web-component-tester/browser.js"></script>
-		<script src="../../web-component-tester/data/a11ySuite.js"></script>
+		<script src="../../@babel/polyfill/browser.js"></script>
+		<script src="../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+		<script src="../../wct-browser-legacy/browser.js"></script>
+		<script src="../../wct-browser-legacy/data/a11ySuite.js"></script>
```

Comment:
Linting complaining about this file cause it doesn't exist.  The tests pass because the modulizer adds `import { a11ySuite } from '../../wct-browser-legacy/a11ySuite.js';` down below, but it doesn't remove this line 😒

Reply:
Looks like we didn't actually need to include this, `browser.js` looks to load it